tigris/collection_test.go

```diff
@@ -240,6 +240,11 @@ func TestCollectionBasic(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, d1, pd)
 
+	mdb.EXPECT().Explain(ctx, "coll_1", driver.Filter(`{"Key1":{"$eq":"aaa"}}`), driver.Projection(nil)).Return(&driver.ExplainResponse{}, nil)
+	explain, err := c.Explain(ctx, filter.Eq("Key1", "aaa"))
+	require.NoError(t, err)
+	require.Equal(t, &driver.ExplainResponse{}, explain)
```

Comment:
Do you want to fill some test data in ExplainResponse?

Reply:
Please also add test on the driver level. It's more important.
So as it tests flow up to the server GRPC handler.
```
driver -> grpc client -> grpc server -> handler
```
for http
```
driver -> http client -> grpc gateway -> grpc server -> handler
```
If ExplainRequest or ExplainResonse requires custom unmarshalling, don't forget to update server/v1/marhsaller.go from the server counterpart.